app/views/Dashboard/Country/index.tsx

```diff
@@ -157,21 +156,26 @@ function Country(props: Props) {
         ),
     );
 
-    const outbreaks = countryResponse?.countryEmergencyProfile.map((item) => {
-        const colors = LINE_COLORS[Number(item.contextIndicatorValue) % LINE_COLORS.length];
-        return (
-            {
-                emergency: item.emergency,
-                fill: colors,
-            }
-        );
+    const outbreaks = unique(
```

Comment:
use useMemo in all of the above functions

Reply:
Done